meditation_app/dom.js

```diff
@@ -3,51 +3,66 @@ const app = () => {
     const play = document.querySelector('.play');
     const outline = document.querySelector('.moving-outline circle');
     const video = document.getElementById('video');
-    const replay = document.querySelector('.replay');
 
-    // Types of sounds
-    const musicTypes = document.querySelectorAll('.misic-type');
     // Timer
-    const timer = document.querySelector('.timer');
+    const minutesArea = document.querySelector('.minutes');
+    const secondsArea = document.querySelector('.seconds')
+
     // Moving-outline length
     const outlineLength = outline.getTotalLength();
-    // Time selector
-    const timeSelectors = document.querySelectorAll('.time-selector__item');
+
     // Time duration
-    let maxDuration = 600;
+    let defaultDuration = 600;
+
+    // Music repetition counter
+    let counter = 0;
 
     outline.style.strokeDasharray = outlineLength;
     outline.style.strokeDashoffset = outlineLength;
 
-    // Change music
-    musicTypes.forEach(musicType => {
-        musicType.addEventListener('click', () => {
-            music.src = musicType.getAttribute("data-sound");
-            video.src = musicType.getAttribute("data-video");
-            checkPlaying(music);
-        });
-    });
-
-    // Play music
-    play.addEventListener('click', () => {
+    function changeMusicType(id) {
+        let typeButton = document.getElementById(id).parentElement;
+        music.src = typeButton.getAttribute("data-sound");
+        video.src = typeButton.getAttribute("data-video");
```

Comment:
why do you need it here? it is not used anywhere. delete it or use in the functions